relay/implementations/beacon-api.ts

```diff
@@ -42,11 +47,13 @@ export class BeaconApi implements IBeaconApi {
 
     const { ssz } = await import('@lodestar/types');
 
-    const beaconBlock = ssz.capella.BeaconBlockBody.fromJson(
+    const sszforkName = (await forkVersionBySlot(slot)) as Capella;
```

Comment:
Why do this cast here?
What if the slot is after capella

Reply:
I changed the function to return `Capella & Deneb` - a type containing only the common field, which fixed the problem.